internal/backends/backend_test.go

```diff
@@ -62,3 +62,91 @@ func TestVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestListDatabasesAndListCollections(t *testing.T) {
```

Comment:
Let's have separate tests for ListDatabases and ListCollections

Reply:
done

Reply:
Seperated and moved the test to appropriate files.